Services/UI/OJS.Services.Ui.Models/Contests/ContestParticipationServiceModel.cs

```diff
@@ -29,7 +31,8 @@ public class ContestParticipationServiceModel : IMapExplicitly
 
     public void RegisterMappings(IProfileExpression configuration)
         => configuration.CreateMap<Participant, ContestParticipationServiceModel>()
-            .ForMember(d => d.Contest, opt => opt.Ignore())
+            .ForMember(d => d.Contest, opt => opt.MapFrom(s => s.Contest))
+            .ForMember(d => d.IsInvalidated, opt => opt.MapFrom(s => s.IsInvalidated))
```

Comment:
No need to map explicitly properties with the same name.

Reply:
If not mapped explicitly, the ```.IsInvalidated``` property will not be mapped at all.

Reply:
Do you know why, it is the same name right (do a copy paste to be sure)?